rake/spec/default/70-st2_actions-integrity_spec.rb

```diff
@@ -6,4 +6,12 @@
   describe command("st2 run packs.install packs=github") do
     its(:exit_status) { is_expected.to eq 0 }
   end
+
+  describe command("st2 run core.local cmd='locale'") do
+    its(:stdout) { should match /UTF-8/ }
+  end
+
+  describe command("st2 run core.local cmd='echo ｈｏｇｅ'") do
+    its(:exit_status) { is_expected.to eq 0 }
```

Comment:
According to build https://circleci.com/gh/StackStorm/st2-packages/1617#tests/containers/0 this fails on [`Debian Wheezy` and `Debian Jessie`](https://github.com/StackStorm/st2-packages/blob/master/circle.yml#L19).

The difference could be that we're using `sysV` init scripts for these 2 platforms, See:
- https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/st2.st2actionrunner.init
- https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/st2.st2actionrunner-worker.init